guide/groovy.md

```diff
@@ -117,79 +117,27 @@ The API can then be invoked by e.g.
-
-<table bgcolor="#ffff00">
-<tr>
-<td>
-    <b>NOTE</b> : From version 3.8.2 the following extra API is available:
-</td>
-</tr>
-</table>
-
-| [Logger](https://www.javadoc.io/doc/org.slf4j/slf4j-api/1.7.30/org/slf4j/Logger.html) getLogger() | This will return the current SLF4J Logger instance. |
-
-
-<table bgcolor="#ffff00">
-<tr>
-<td>
-    <b>NOTE</b> : From version 4.0 the following extra API is available:
-</td>
-</tr>
-</table>
-
-| [FileIO](https://github.com/release-engineering/pom-manipulation-ext/blob/master/io/src/main/java/org/commonjava/maven/ext/io/FileIO.java) getFileIO() | This will return a FileIO instance for remote File resolving. |
-| [PomIO](https://github.com/release-engineering/pom-manipulation-ext/blob/master/io/src/main/java/org/commonjava/maven/ext/io/PomIO.java) getPomIO() | This will return a PomIO instance for parsing POM models. |
-| [Translator](https://github.com/release-engineering/pom-manipulation-ext/blob/master/io/src/main/java/org/commonjava/maven/ext/io/rest/Translator.java) getRESTAPI() throws [ManipulationException](https://github.com/release-engineering/pom-manipulation-ext/blob/master/common/src/main/java/org/commonjava/maven/ext/common/ManipulationException.java) | Gets a configured VersionTranslator to make REST calls to DA. |
-
-<table bgcolor="#ffff00">
-<tr>
-<td>
-    <b>NOTE</b> : From version 4.4 the Translator interface has changed from providing <code>translateVersions</code> to:
-
-    <table>
-        <tr>
-        <td><code>lookupVersions</code></td>
-        <td>Lookup versions respecting DA suffix priority schemes which will return the best matched version</td>
-        </tr>
-        <tr>
-        <td><code>lookupProjectVersions</code></td>
-        <td>Lookup versions (e.g. for a project) ignoring DA suffix priority schemes returning the latest version</td></tr>
-    </table>
-</td>
-</tr>
-</table>
+| Method | Description | Since      |
+| -------|:------------|:-----------|
+| [Map](https://docs.oracle.com/javase/7/docs/api/java/util/Map.html)<[ProjectVersionRef](https://github.com/Commonjava/atlas/blob/master/identities/src/main/java/org/commonjava/atlas/maven/ident/ref/ProjectVersionRef.java), [String](https://docs.oracle.com/javase/7/docs/api/java/lang/String.html)> translateVersions([List](https://docs.oracle.com/javase/7/docs/api/java/util/List.html)<[Project](https://github.com/release-engineering/pom-manipulation-ext/blob/master/common/src/main/java/org/commonjava/maven/ext/common/model/Project.java)>) | Translate the versions. _Removed in 4.3; use `lookupProjectVersions` or `lookupVersions` instead_. | 1.7 |
```

Comment:
This is an interesting one - how to deal with _removed_ API. I am not sure whether its better to have it in or remove it. Or use 'strike-out' to emphasise its no longer available?

Reply:
Strike out the name maybe? I also thought to write a range for since like `[1.7, 4.3]`.

Reply:
Strike-out seems reasonable and you could add the range for the 'since' as well ?

Reply:
Sure. I can do both.